coalib/bearlib/aspects/Metadata.py

```diff
@@ -219,3 +219,23 @@
         """
     )
 )
+
+Root.Metadata.CommitMessage.Body.new_subaspect(
+    "Existence",
+    AspectDocumentation(
+        definition="""
+        Forces the commit message body to exist (nonempty).
+        """,
+        example="""
+        aspects: Add CommitMessage.Body
+        """,
+        example_language="English",
+        importance_reason="""
+        Having a nonempty commit body is important if you consistently want
+        elaborate documentation on all commits.
+        """,
+        fix_suggestions="""
+        Write a commit message with a body.
+        """
+    )
```

Comment:
Description should be unbiased - `Describes whether the commit body exists or is empty` or something like that IMO